### DISCUSSION TITLE
messaging

### DISCUSSION THREAD

**Discussion by Calvaxe** [Chatterino/chatterino2#3957] [Q&A] [answered]
---

**Comment by pajlada** [comment, marked as answer]
There's no way to control that from our side - we just send a message and Twitch deals with it and prioritizes it.

**Reply by LosFarmosCTL** [reply]
They are using verified bot accounts, messages from those do not get dropped silently by twitch, thus they can always send them even in high-pace chats.

https://dev.twitch.tv/docs/irc#verified-bots
